### ISSUE TITLE
bull-board(v1.0.0-alpha.0) getting errors while testing the alpha, might be doing something wrong?

### ISSUE THREAD

**Issue by arkhamRejek** [vcapretz/bull-board#65]
```javascript
import {  router } from 'bull-board';

app.use('/ui', router)

```

trace:
`
 at Function.render (/Users/dev/platform/node_modules/express/lib/application.js:580:17)
    at ServerResponse.render (/Users/dev/platform/node_modules/express/lib/response.js:1012:7)
    at exports.entryPoint (/Users/dev/platform/node_modules/bull-board/src/routes/index.ts:8:7)
    at Layer.handle [as handle_request] (/Users/dev/platform/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/dev/platform/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/dev/platform/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/dev/platform/node_modules/express/lib/router/layer.js:95:5)
    at /Users/dev/platform/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/dev/platform/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/dev/platform/node_modules/express/lib/router/index.js:275:10)
`

**Comment by philefstat** [comment]
Along with a similar trace I'm getting the error message `Error: Failed to lookup view "index" in views directory ...`

**Comment by jamesholcomb** [comment]
Same error as @philefstat { Error: Failed to lookup view "index" in views directory "/Users/jamesholcomb/code/myapp/node_modules/bull-board/dist/../src/ui"

**Comment by vcapretz** [comment]
Hey folks! Thank you for testing the alpha, sorry for the bugs tho 😅
Is it still happening in the v1.0.0-alpha.1 version?

**Comment by jamesholcomb** [comment]
v1.0.0-alpha.0

**Comment by arkhamRejek** [comment]
same issues with v1.0.0-alpha.1

**Comment by Deadly0** [comment]
It is possible to use bull-board with bullmq right now? Have the same issue with alpha1. Is the stable version works with bullmq?

**Comment by vcapretz** [comment]
we are going to take a look at why is this problem happening and make another alpha version available! sorry for the delay 😄 

@Deadly0 the latest stable 0.6 does not support bullmq unfortunately 👍

**Comment by luastoned** [comment]
`Error: Failed to lookup view "index" in views directory "/usr/src/app/node_modules/bull-board/dist/../src/ui"` seems like something isn't getting compiled properly, hence the `.../src`

**Comment by hieptuanle** [comment]
The problem is that the `index.ejs` file from `/src/ui/index.ejs` is not copied to the `dist` folder when the package was built.

Create a duplicate of this file https://github.com/vcapretz/bull-board/blob/master/src/ui/index.ejs in `/dist/ui` should fix the problem.

**Comment by vcapretz** [comment]
Hey folks, sorry if I take too long to review/comment in the issues 😓 

btw, I just released `v1.0.0-alpha.5`, for what I could test it works now! 

let me know if this particular bug is solved and if you find any others please start new threads 😄

**Comment by vcapretz** [comment]
closing for now, please open new issues if you find any other problems 😄
